content/en/security/cloud_siem/_index.md

```diff
@@ -51,33 +51,202 @@ further_reading:
-{{< /whatsnext >}}
+If you don't already have a Datadog account, sign up for a [free trial][2]. After you log in to your Datadog account:
+
+1. Navigate to [Cloud SIEM][3].
+1. Click **Enable Cloud SIEM**.
+1. Follow the onboarding steps.
+
+See the [Getting Started Guide][4] for more detailed setup instructions.
+
+## Key features
+
+### Security and observability
+
+Cloud SIEM embeds both cloud and on-premises telemetry directly into security workflows to accelerate investigation and response. And with a shared platform that brings DevOps and Security teams together, organizations can break down silos and respond to threats collaboratively and efficiently.
+
+### Flexible cost control
+
+As your organization scales, controlling the ingestion cost of security logs without compromising visibility is critical. Cloud SIEM is integrated with Datadog Log Management so you can choose the retention and querying capability of your security logs to balance cost efficiency and your threat detection needs. The available options are:
+- [Standard indexing][6] for logs that need to be queried frequently and retained short term.
+- [Flex Logs][7] for logs that need to be retained long-term, but sometimes need to be queried urgently.
+- [Log Archives][8] for logs that are infrequently queried and need to be stored long-term, such as audit logs.
+
+## Cloud SIEM Overview page
+
+Navigate to the [Cloud SIEM Overview dashboard][2]. Use this dashboard to see key security insights and act on common workflows for security analysts, security and detection engineers, and Security Operations Center (SOC) managers. From the Overview page, you can:
+- Access important signals, open cases, and high-risk entities.
+- Complete onboarding tasks and review content‑pack health.
+- View and investigate top signals by geography or ISP.
+- Analyze signals and rules by MITRE ATT&CK tactics.
+- Track detection performance (MTTD, false‑positive rates).
+- Read the latest [Security Labs][1] research and release notes.
+
+Click **Customize Page** to reorder or hide modules so you can see what is important to you.
+
+Learn more about each Cloud SIEM Overview page section below.
+
+### Security coverage
+
+{{< img src="security/security_monitoring/landing/01_security_coverage.png" alt="Security coverage sections showing 11 active and 1 broken content packs and a bar graph of logs analyzed by Cloud SIEM" style="width:100%;" >}}
+
+Remain aware of any data processing issues or coverage gaps.
+
+#### Enabled content packs and integrations
+
+View enabled content packs and integrations across the critical categories for providing comprehensive security coverage. Hover over each section of the horizontal bar to see which content packs are enabled in each category.
+
+#### Content pack and logs health KPIs
+
+See whether any content packs or integrations are in warning or broken states so that you can quickly resolve any coverage gaps. Click a status tile to view the affected content packs.
```

Comment:
```suggestion
See whether any content packs or integrations are in warning or broken states so that you can resolve any coverage gaps. Click a status tile to view the affected content packs.
```